frontend/src/metabase/containers/ItemPicker/ItemPicker.unit.spec.js

```diff
@@ -117,6 +117,11 @@ function mockCollectionItemsEndpoint() {
       data,
     };
   });
+
+  fetchMock.get("path:/api/collection/personal/items", {
```

Comment:
personal collection items weren't mocked, what led to the error

Reply:
The same - please make sure we user helpers from `server-mocks`